kafka-connect-s3/src/test/java/io/confluent/connect/s3/integration/README.md

```diff
@@ -9,4 +9,8 @@ the `.aws/credentials` file.
 
 ## Running the Tests
 Tests can be run manually using the `mvn integration-test` command 
-or through IntelliJ by running the class.
\ No newline at end of file
+or through IntelliJ by running the class.
+
+If running in IntelliJ, temporarily disable the network ITs in by
```

Comment:
See my comment above. 
That's an annoying hoop to jump

Reply:
Agree